### DISCUSSION TITLE
Audio track defaults to wrong language

### DISCUSSION THREAD

**Discussion by grizzlybeartist** [FreeTubeApp/FreeTube#6287] [Q&A]
Is there any way to fix this? Every time I watch a video with multiple audio tracks, it defaults to literally anything but English, usually German. My location is set to US in my settings, my vpn is set to US too, so it's not something on my end. Is there any way to set it so the audio track is always the default one? Even if I _could_ understand what it was saying, the crappy ai voiceover is irritating.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Please read: https://github.com/FreeTubeApp/FreeTube/issues/4291#issuecomment-2452527942

**Reply by bulletbaillie** [reply]
I also have this isssue since the latest v0.23.3 release, and have been unable to find a solution.
